### DISCUSSION TITLE
Alternative way to edit read only note

### DISCUSSION THREAD

**Discussion by tanandry** [TriliumNext/Trilium#1403] [Q&A] [answered]
First of all, thanks for making this great app 👍🏻 

I would like to know whether there is another way to edit read only note besides clicking on the "Edit" button at the very beginning of a note? The reason is I frequently forget to click on it, scroll down quite far away, when I want to edit I just remember and have to go back to the top again. This is quite bothersome.

Why not make it so when I double click on anywhere inside the note then it is editable? Or perhaps a keyboard shortcut?

Thanks 😄

**Comment by zadam** [comment, marked as answer]
Your proposed solution would still not solve your issue - keeping the scrolling position. The key problem is that read only and editable views are fundamentally different - former is just HTML view, latter is editing component loaded with HTML. This is intentional since one purpose of read only notes is to keep decent performance for huge notes (which otherwise take long time to load into editor).

**Reply by tanandry** [reply]
Ah.. I didn't even think about keeping the scrolling position, but you're right. Though if there is an alternative shortcut at least I don't need to scroll back up again just to click the button, that part will be done automatically. But that's fine, thanks 👍🏻
